apps/src/music/views/MusicView.jsx

```diff
@@ -346,6 +346,12 @@ class UnconnectedMusicView extends React.Component {
       !!levelData?.text || !!this.props.longInstructions
     );
 
+    const startSources = this.getStartSources();
+    const predictSettings = this.props.levelProperties?.predictSettings;
```

Comment:
Nit: Since `predictSettings` and `isPredictLevel` is used in branch below, maybe easier to set these variables below (line 370)?

Reply:
Good call!